DAS/das/tests/mqtt_test.py

```diff
@@ -29,19 +31,20 @@
 
 def start_publishing(client, args):
     print("start")
-    client.publish("power_model/start")
-    client.publish("start")
+    client.publish(str(BOOST.start))
```

Comment:
Why do we need to convert these to string, aren't they already a string?

Reply:
They're not, they're actually our `Topic` class. Having a class means we can do fancy stuff to build new Topics and compare them but unfortunately we do at some point need to convert to a string.

Reply:
wait so the `start` attribute in `BOOST` is a class?

Reply:
Sorry probably should have said! That was one of the changes in monash-human-power/common#12. See [here](https://github.com/monash-human-power/common/blob/6dd9bd5bcb8395f6e88b6364e568884fc8eb18a1/mhp/topics/topics.py#L60).

Reply:
Oh I see, interesting implementation!